### DISCUSSION TITLE
Downloading CBC with svn

### DISCUSSION THREAD

**Discussion by kish-shen** [coin-or/Cbc#493] [Q&A]
We use our own script to download and build CBC, and I noticed that we
are still using svn to download CBC, even though you have switched to
using git.

For example, to download CBC 2.10, we use the following svn command:

svn co https://projects.coin-or.org/svn/Cbc/stable/2.10

Is this still the best way to download the stable released of a
specific version of CBC?

Thanks,

Kish

**Comment by tkralphs** [comment]
The svn repo has not received any updates since we switched to git years ago. To download and build Cbc from source, just follow the detailed instructions in the [README](https://github.com/coin-or/Cbc/blob/master/README.md), which also gives pointers to additional documentation.
